android/app/src/main/java/com/mattermost/rnbeta/MattermostManagedModule.java

```diff
@@ -0,0 +1,70 @@
+package com.mattermost.rnbeta;
+
+import android.app.Application;
+import android.content.Context;
+import android.os.Bundle;
+
+import com.facebook.react.bridge.Arguments;
+import com.facebook.react.bridge.NativeModule;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+import com.facebook.react.bridge.Promise;
+import com.facebook.react.modules.core.DeviceEventManagerModule;
+
+public class MattermostManagedModule extends ReactContextBaseJavaModule {
+    public static MattermostManagedModule instance;
```

Comment:
This should probably be private. You have a public method, `getInstance(...)` declared on line 28 that returns one of these. By leaving this public, it looks like theres a possibility that somebody might get an uninitialized `MattermostManagedModule`